backend/lib/endpoints/feedback.js

```diff
@@ -0,0 +1,36 @@
+const httpErrors = require("http-errors");
+const { createFeedbackSchema } = require("./schema/feedback");
+
+/*
+ * /api/feedback
+ */
+async function routes(app) {
+  const Feedback = app.mongo.model("Feedback");
+
+  app.post("/", { schema: createFeedbackSchema }, async (req) => {
+    const { userId } = req.body;
+    let ip = "undefined";
+    if (userId) {
+      const userFeedback = await Feedback.find({ userId });
+      if (userFeedback[0]) {
+        return new httpErrors.Conflict("User already submitted feedback");
+      }
+    }
+
+    if (req.ip !== null && typeof req.ip !== "undefined") {
+      ip = req.ip;
+      const feedbackFromIp = await Feedback.find({ ipAddress: ip });
+      if (feedbackFromIp[0]) {
+        return new httpErrors.Conflict(
+          "Feedback already submitted with this IP address",
+        );
+      }
+    }
+    return new Feedback({
+      ...req.body,
+      ipAddress: ip,
+    }).save();
+  });
```

Comment:
Was talking with @robinv85 at the posts PR if we shouldn't be setting the response code to `201 - Created` in case of success here

Reply:
Shouldn't? It is currently a `200 OK` response.

Reply:
yeah 201 would be better indeed